src/App.tsx

```diff
@@ -1,14 +1,18 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { Todo } from './types/Todo';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
 export const App: React.FC = () => {
+  const [todoModal, setTodoModal] = useState<Todo | null>(null);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todosAreLoaded, setTodosAreLoaded] = useState(false);
```

Comment:
we usually name boolean vars starting with is
```suggestion
  const [isTodosAreLoaded, setIsTodosAreLoaded] = useState(false);
```